Kelvin_Lau_Project1.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Kelvin_Lau_Project1</title>
```

Comment:
You are missing some essential meta tags such as charset and viewport. Review to the HTML lesson and the responsive CSS lesson for more info!